robocop/checkers/naming.py

```diff
@@ -751,3 +775,11 @@ def check_if_keyword_is_deprecated(self, keyword_name, node):
                 col=col,
                 version=f"{ROBOT_VERSION.major}.*",
             )
+
+    def visit_LibraryImport(self, node):  # noqa
+        if ROBOT_VERSION.major < 5 or (ROBOT_VERSION.major == 5 and ROBOT_VERSION.minor == 0):
```

Comment:
This can be replaced with ``ROBOT_VERSION < RF_5_1`` code after official release - version package doesn't work with comparing the boundaries of beta version (5.1beta is not considered to be 5.1).